.coderabbit.yaml

```
@@ -1,212 +1,114 @@
-  # 성능 관련 패턴
-  performance_patterns:
-    - pattern: "for\\s*\\(.*\\s*;\\s*.*\\.length\\s*;.*\\)"
-      message: "반복문에서 매번 length를 계산하고 있습니다. 길이를 변수에 저장하여 성능을 개선하세요."
-      severity: "low"
+# ============================================================================
+# 보안 검사 설정
+# ============================================================================
+security:
+  secrets_detection: true    # 하드코딩된 시크릿 (API 키, 패스워드 등) 탐지
+  dependency_check: true     # 의존성 라이브러리 보안 취약점 검사
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)